### PR TITLE
Moebius nerfs: Gun mods

### PR DESCRIPTION
**Pull request by MadmanMartian**
## About The Pull Request

Moves the research gun mods to their own file

Nerfs the Shunt to +50% damage, rather than double damage

Removes the Honk trigger

## Why It's Good For The Game

The shunt, coupled with the laser cannon and a charge attack, could one-tap anyone not in high tier armor.

The honk noise was added as an easter egg for using the clown horn on anything, with the tradeoff being worse stats in every way on the tool/gun. Adding it to a mass-producable tool/weapon mod defeated the purpose.

## Changelog
:cl:
del: The honk trigger has been removed.
balance: The shunt now only gives an additional 50% damage
/:cl:

### CODE REVIEW

**Review by mattroks101 — APPROVED** [review]

**Review by TorinoFermic — APPROVED** [review]

File: code/modules/projectiles/guns/mods/research_mods.dm [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	matter = list(MATERIAL_PLASTIC = 2, MATERIAL_PLASTEEL = 1, MATERIAL_GOLD = 1, MATERIAL_URANIUM = 1)
+	spawn_blacklisted = TRUE
+
+/obj/item/weapon/gun_upgrade/mechanism/battery_shunt/New()
+	..()
+	var/datum/component/item_upgrade/I = AddComponent(/datum/component/item_upgrade)
+	I.weapon_upgrades = list(
+	GUN_UPGRADE_RECOIL = 1.2,
+	GUN_UPGRADE_DAMAGE_MULT = 1.5,
+	GUN_UPGRADE_CHARGECOST = 3)
```
Comment (TheShown911) [inline code/modules/projectiles/guns/mods/research_mods.dm:62]: I'd put this down to 2, would still be a nerf overall.

**Comment by basedcringe** [comment]
I have no horn and yet I must honk

**Review by genessee596 — COMMENTED** [review]
good

**Review by genessee596 — APPROVED** [review]

**Comment by TheShown911** [comment]
The removal of the honk trigger and it's consequences have been a disaster on the spees race.

**Pull request merged**